runtime/compiler/optimizer/TreeLowering.cpp

```diff
@@ -48,17 +49,14 @@ TR::TreeLowering::perform()
       TR::Node* node = nodeIter.currentNode();
       TR::TreeTop* tt = nodeIter.currentTree();
 
-      if (TR::Compiler->om.areValueTypesEnabled())
```

Comment:
Is there a reason the test of `areValueTypesEnabled()` needs to be removed?  I think the transformations that are being applied will only be needed if value types support is enabled.

Reply:
`areValueTypesEnabled()` is already checked at the beginning of the method. It's redundant to check it again here. 
https://github.com/eclipse/openj9/blob/c15c9f860a94feb2a22b245c1e5a33bf16a75818/runtime/compiler/optimizer/TreeLowering.cpp#L37-L43